lib/heap/heap.h

```diff
@@ -244,7 +244,7 @@ static inline chunksz_t min_chunk_size(struct z_heap *h)
 
 static inline size_t chunksz_to_bytes(struct z_heap *h, chunksz_t chunksz_in)
 {
-	return chunksz_in * CHUNK_UNIT - chunk_header_bytes(h);
+	return (chunksz_in * CHUNK_UNIT) - chunk_header_bytes(h);
```

Comment:
Again, this is the exact counter example in https://wiki.sei.cmu.edu/confluence/display/c/EXP00-C.+Use+parentheses+for+precedence+of+operation